thunderbird/utils.py

```diff
@@ -114,5 +115,6 @@ def log_handler(process, response, message, process_step=None):
         status_percentage = response.status_percentage
 
     # Log to all sources
-    logger.info(message)
+    pywps_logger.info(message)
+    stderr_logger.critical(message)
```

Comment:
How come the logging levels are different for these sources?

Reply:
I have tested several logging levels, and it seems like the level `INFO` is too low that restrains the logging message to be printed out in terminal while `CRITICAL` ensures the log messages to be printed out.

Reply:
I think hardcoding the log levels was a mistake on my part.  We would want to use something like this, `logger.log(level, message)`.  We can default the loglevel to `info` and define it as a param.

Logging a message as `critical` probably isn't the best idea.  It puts a huge amount of significance onto the logging message that may or may not actually be critical.  We want to ensure we are using the appropriate levels.